test/widgets/test_widget_init_configure.py

```diff
@@ -59,6 +59,7 @@
 # To skip a test entirely, list the widget class here
 no_test = [widgets.Mirror, widgets.PulseVolume]  # Mirror requires a reflection object
 no_test += [widgets.ImapWidget]  # Requires a configured username
+no_test += [widgets.Redshift]  # Initialisation will reset redshift
```

Comment:
What does `initialisation will reset redshift` mean? This should be fine to test if we make the changes so the widget displays an error if it can't run `redshift`.

Reply:
The widget now displays a crash error, I've also allowed the initialisation test to take place which should now be fine.